lib/dwh/dml/merge.go

```diff
@@ -39,7 +60,7 @@ func MergeStatement(fqTableName, subQuery, pk, idempotentKey string, cols []stri
 	return fmt.Sprintf(`
 			MERGE INTO %s c using (%s) as cc on c.%s = cc.%s
 				when matched AND cc.%s then DELETE
-				when matched AND IFNULL(cc.%s, false) = false %s then UPDATE
+				when matched AND IFNULL(cc.%s, false) = false %sthen UPDATE
```

Comment:
This is on purpose. This is to avoid double spacing since the idempotent clause has spacing already.